specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/examples/ManagedEnvironments_Get.json

```diff
@@ -19,7 +19,21 @@
           "deploymentErrors": null,
           "internalLoadBalancerEnabled": false,
           "defaultDomain": "jlaw-demo1.k4apps.io",
-          "staticIp": "20.42.33.145"
+          "staticIp": "20.42.33.145",
+          "certificates": [
+            {
+              "password":"private key password",
+              "name":"certificate-firendly-name",
+              "subjectName":"my.custom.subject.name.example",
+              "value" :"PFX-or-PEM-blob"
```

Comment:
We did agree to have separate APIs but I don't think we specifically said NOT to include certificates when GETting the Managed Environment (since they are child resources).

Having said that I think it simplifies things if we don't return it (or create/update them) together as part of the Managed Environment. Will make an update

Reply:
It is generally not recommended to include the state of the child resources in the parent resource. The resources are cached in the Azure Resource Graph and get updated there when ARM API calls are made. Having the certs collection in the ManagedEnvironment object will make the object inconsistent in Azure Resource Graph when a certificate collection is updated via separate APIs.